web/src/components/Nav.jsx

```diff
@@ -1,8 +1,6 @@
 import React from 'react';
-import {
-  NavLink
-} from "react-router-dom";
-import { fade, makeStyles } from '@material-ui/core/styles';
+import { NavLink } from "react-router-dom";
```

Comment:
from what I can tell this is the part of the code that is causing the merge conflict

Reply:
Resolved with new commit merging current sprint1-branch into this branch and updating and conflicts to match as necessary